salt/repo/online.sls

```diff
@@ -0,0 +1,36 @@
+{%- from "repo/map.jinja" import repo with context %}
+
+{%- if grains["os_family"] == "RedHat" %}
+
+Configure EPEL repository:
+  pkg.installed:
+  {%- if grains["os"] == "CentOs" %}
+    - name: epel-release
+  {%- else %}
+    - sources:
+      - epel-release: https://dl.fedoraproject.org/pub/epel/epel-release-latest-{{ grains['osmajorrelease'] }}.noarch.rpm
+  {%- endif %}
+
+  {%- if grains["os"] == "RedHat" %}
+check that the system is registered:
```

Comment:
Why?

Reply:
It's just a check as we need the system to be registered on RedHat to be able to install online stuff but maybe it's a requirement and we should remove this from the state it's ok for me

Reply:
Nah, I've seen this on other prereq lists we have, so if there's a use for this, can stay as far as I'm concerned. I trust your judgement.